2022-04-18 17:42:54,015 INFO    MainThread:111481 [wandb_setup.py:_flush():71] setting env: {}
2022-04-18 17:42:54,027 INFO    MainThread:111481 [wandb_setup.py:_flush():71] setting login settings: {}
2022-04-18 17:42:54,028 INFO    MainThread:111481 [wandb_init.py:_log_setup():371] Logging user logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_174253-3jfr0r2p/logs/debug.log
2022-04-18 17:42:54,029 INFO    MainThread:111481 [wandb_init.py:_log_setup():372] Logging internal logs to /scratch/users/fsofian19/COMP491_model/models/ast_custom/egs/custom/wandb/run-20220418_174253-3jfr0r2p/logs/debug-internal.log
2022-04-18 17:42:54,030 INFO    MainThread:111481 [wandb_init.py:init():404] calling init triggers
2022-04-18 17:42:54,031 INFO    MainThread:111481 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-04-18 17:42:54,031 INFO    MainThread:111481 [wandb_init.py:init():460] starting backend
2022-04-18 17:42:54,032 INFO    MainThread:111481 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-18 17:42:54,069 INFO    MainThread:111481 [backend.py:ensure_launched():216] starting backend process...
2022-04-18 17:42:54,088 INFO    MainThread:111481 [backend.py:ensure_launched():221] started backend process with pid: 111542
2022-04-18 17:42:54,096 INFO    MainThread:111481 [wandb_init.py:init():469] backend started and connected
2022-04-18 17:42:54,105 INFO    MainThread:111481 [wandb_init.py:init():533] updated telemetry
2022-04-18 17:42:54,189 INFO    MainThread:111481 [wandb_init.py:init():563] communicating current version
2022-04-18 17:42:59,191 INFO    MainThread:111481 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-04-18 17:43:03,004 INFO    MainThread:111481 [wandb_init.py:init():606] starting run threads in backend
2022-04-18 17:43:08,032 INFO    MainThread:111481 [wandb_run.py:_console_start():1810] atexit reg
2022-04-18 17:43:08,035 INFO    MainThread:111481 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-04-18 17:43:08,037 INFO    MainThread:111481 [wandb_run.py:_redirect():1689] Redirecting console.
2022-04-18 17:43:08,042 INFO    MainThread:111481 [wandb_run.py:_redirect():1745] Redirects installed.
2022-04-18 17:43:08,043 INFO    MainThread:111481 [wandb_init.py:init():633] run started, returning control to user process
2022-04-18 17:48:17,637 INFO    MainThread:111481 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-04-18 17:48:17,643 INFO    MainThread:111481 [wandb_run.py:_restore():1752] restore
2022-04-18 17:48:20,484 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 17:48:20,905 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 17:48:21,184 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 1734
}

2022-04-18 17:48:21,590 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 17:48:21,694 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 17:48:21,799 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 17:48:21,904 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1734
  total_bytes: 26493
}

2022-04-18 17:48:22,012 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:22,119 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:22,224 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:22,328 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:22,433 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:22,537 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:23,526 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}

2022-04-18 17:48:24,107 INFO    MainThread:111481 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26493
  total_bytes: 26493
}
local_info {
}

2022-04-18 17:48:27,485 INFO    MainThread:111481 [wandb_run.py:_append_files():2180] logging synced files
